sky/resources.py

```diff
@@ -370,7 +371,7 @@ def _set_region_zone(self, region: Optional[str],
         self._region, self._zone = self._cloud.validate_region_zone(
             region, zone)
 
-    def get_offering_regions_for_launchable(self) -> List[clouds.Region]:
+    def get_valid_regions_for_launchable(self) -> List[clouds.Region]:
         """Returns a set of `Region`s that can provision this Resources.
```

Comment:
How about adding a para: (Internal) This function respects any config in `skypilot_config` that may have restricted the regions to be considered (e.g., a `ssh_proxy_command` dict with region names as keys).

Reply:
Added. Thanks!